src/main/java/seedu/taassist/model/student/Address.java

```diff
@@ -9,10 +9,10 @@
  */
 public class Address {
 
-    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, but they should not be blank";
 
     /*
-     * The first character of the address must not be a whitespace,
+     * The first character of the session must not be a whitespace,
```

Comment:
```suggestion
     * The first character of the address must not be a whitespace,
```

Reply:
And I can't mark line 34 but maybe it should be `     * Returns true if a given string is a valid address.`